test/unit/three_scale/swagger/specification_test.rb

```diff
@@ -13,6 +13,16 @@ class ThreeScale::Swagger::SpecificationTest < ActiveSupport::TestCase
     assert specification.valid?, 'specification should be valid'
   end
 
+  test "invalid 2.0 spec errors" do
+    specification = fixture_spec('invalid-2.0')
+    assert_not specification.valid?
+    errors = specification.errors.full_messages_for(:base).join(', ')
```

Comment:
if these are separate messages that you match in full, I assume no need to `join` them. If you partially match them, this is fine though.

Reply:
Yeah, I only want to match a part of the message, because it's too long. There is an example of the full message in https://github.com/3scale/porta/pull/3934/commits/07737ed3492c0b689586b1c230739e3892eef8ae#r1839063674
I don't think we need to be that exact.